Dockerfile

```diff
@@ -10,11 +10,15 @@ RUN useradd $username -u $user_id --create-home --user-group
 
 COPY ./dist/suzieq-$version-py3-none-any.whl  /tmp/
 
-RUN pip install --upgrade pip
-RUN pip install /tmp/suzieq-$version-py3-none-any.whl
-RUN rm /tmp/suzieq-$version-py3-none-any.whl
 COPY --chown=$username suzieq/config/etc/suzieq-cfg.yml /home/$username/.suzieq/suzieq-cfg.yml
-RUN sed -i 's/127.0.0.1/0.0.0.0/' /home/$username/.suzieq/suzieq-cfg.yml
+
+RUN python3 -m pip install --upgrade pip && \
```

Comment:
Great question.  It was [officially recommended in the python docs](https://docs.python.org/3/installing/index.html#basic-usage) as the way to install back in 2014.

There are a number of articles about it:

- https://stackoverflow.com/questions/25749621/whats-the-difference-between-pip-install-and-python-m-pip-install
- https://www.reddit.com/r/learnpython/comments/mqsp3u/python_m_pip_install_vs_pip_install/
- https://snarky.ca/why-you-should-use-python-m-pip/

Ultimately it will make no difference here.  Its just habit, and I am happy to revert.

Reply:
As far as I understood the main reason for using `python3 -m pip` is to have more control on the version of python we are using and to have a command that works even when the environment is not correctly configured. But since we are in a container, which is a controlled environment, where we have a single version of python I would go with just `pip`, it's shorted and if we have a problem with it, the problem is not using just `pip` but the way we are building the container.